go-kit/metrics/provider/otel/provider.go

```diff
@@ -128,7 +130,14 @@ func New(ctx context.Context, serviceName string, opts ...Option) (xmetrics.Prov
 
 // Start starts the provider's controller and exporter.
 func (p *Provider) Start() error {
-	return nil
+	var err error
+	if p.cfg.enableRuntimeMetrics {
+		err = runtime.Start(
+			runtime.WithMeterProvider(p.meterProvider),
+			runtime.WithMinimumReadMemStatsInterval(p.cfg.collectPeriod),
```

Comment:
Isn't this interval configurable? defaults seem to be 1 minute (`collectPeriod` has the `DefaultReaderInterval`)

Reply:
Correct, it uses the default, unless you set the `WithCollectPeriod` option.

https://github.com/heroku/x/blob/master/go-kit/metrics/provider/otel/options.go#L47-L53